tests/integ/modin/test_telemetry.py

```diff
@@ -513,6 +514,44 @@ def test_telemetry_property_iloc(
     ]
 
 
+def _set_iloc(df: pd.DataFrame) -> None:
+    df.iloc = 3
+
+
+def _del_iloc(df: pd.DataFrame) -> None:
+    del df.iloc
+
+
+@pytest.mark.parametrize(
+    "method_verb, name, method, method_noun",
+    [("set", "iloc", _set_iloc, "setter"), ("delete", "iloc", _del_iloc, "deleter")],
+)
+def test_telemetry_property_missing_methods(method_verb, name, method, method_noun):
```

Comment:
Why is set and delete labeled as missing methods? I'm not sure yet why this isn't able to be combined with test_telemetry_property_iloc test